appserver/admin/admin-core/pom.xml

```diff
@@ -56,12 +58,12 @@
 
     <dependencies>
         <dependency>
-            <groupId>fish.payara.server.internal.admin</groupId>
+            <groupId>fish.payara.server.core.admin</groupId>
             <artifactId>admin-util</artifactId>
             <version>${project.version}</version>
```

Comment:
The harder part of this is to actually utilize the bom -- have it in dependency management next to payara-bom and remove all the versions from these places. (in regex speak, when groupId with core is followed by version tag, remove the version tag ;))